### ISSUE TITLE
Ambiguity of the ternary operator with ??

### ISSUE THREAD

**Issue by tamashalasi** [trustedtomato/proposal-partial-expression#4]
The code `#?? +1 : 2` needs a lookahead for the parsing, and that lookahead might be at any size, since  the code could be `#?? +11111111111111 : 2` too.
Any idea for another notation?
Or is there a need for them at all?

*Mike Samuel [mentioned](https://esdiscuss.org/topic/partial-expression-proposal#content-1) the bug.*

**Comment by tamashalasi** [comment]
There are really not obvious cases which suggest that the referring to outer layers should be eliminated:
```javascript
// Example 1:
let foo = #foo(#???:??)

// Example 2:
let constant = ##??

// Example 3:
let makeAdder = ##?+??
```

*Credit goes to Isiah Meadows. He [mentioned](https://esdiscuss.org/topic/partial-expression-proposal#content-3) these.*

**Comment by tamashalasi** [comment]
Resolved in fa0f074202dd8b46f14b052ec92289dffe7cc4c3.
